doc/api/url.md

```diff
@@ -608,6 +608,53 @@ console.log(JSON.stringify(myURLs));
 // Prints ["https://www.example.com/","https://test.example.org/"]
 ```
 
+#### `URL.createObjectURL(blob)`
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `blob` {Blob}
+* Returns: {string}
+
+Creates a `'blob:nodedata:...'` URL string that represents the given {Blob}
+object and can be used to retrieve the `Blob` later.
+
+```js
+const {
+  Blob,
+  resolveObjectURL,
+} = require('buffer');
+
+const blob = new Blob(['hello']);
+const id = URL.createObjectURL(blob);
+
+// later...
+
+const otherBlob = resolveObjectURL(id);
+console.log(otherBlob.size);
+```
+
+The data stored by the registered {Blob} will be retained in memory until
+`URL.revokeObjectURL()` is called to remove it.
+
+`Blob` objects are registered within the current thread. If using Worker
+Threads, `Blob` objects registered within one Worker will not be available
+to other workers or the main thread.
```

Comment:
Hmm, shouldn't it be available to other threads?

Reply:
I don't believe so. In browsers, the created object URL is scoped to ... "the document in the window on which it was created". While there are definite differences here, I think limiting the create object URL in Node.js to the creating Environment is semantically the closest match.

Reply:
ok, so if i do want to share something with another thread, then i do need to postMessage the blob.

Reply:
Yeah, and given the way Blobs are shared we ensure that the data is shared efficiently.

Reply:
Just to clarify, they are cross thread in web specs. https://www.w3.org/TR/FileAPI/#requestResponseModel , dereferencing is based upon the entire User Agent, only the lifetime is affected by a Realm being cleaned up ( https://www.w3.org/TR/FileAPI/#lifeTime )